packages/noya-state/src/reducers/canvasReducer.ts

```diff
@@ -223,6 +257,230 @@ export function canvasReducer(
 
             break;
           }
+          case 'movingPoint': {
+            const { current, selectedPoint } = interactionState;
+            let delta = { x: 0, y: 0 };
+            const boundingRects = getBoundingRectMap(
+              getCurrentPage(state),
+              Object.keys(state.selectedPointLists),
+              {
+                clickThroughGroups: true,
+                includeArtboardLayers: false,
+                includeHiddenLayers: false,
+              },
+            );
+            //Find point to measure distance
```

Comment:
I feel like there could be a better way to calculate the distance between drag start and drag end so we don't have to pass a selectedPoint through the action, any ideas?

Reply:
We can try to use `origin` and `pageSnapshot`, like we do for moving layers.

This approach is potentially safer anyway, in case we encounter a degenerate case where the layer's frame has width/height of 0 after computing the tight bounds, since we scale the points by the layer's frame. I'm not sure if that can occur - it can't occur when moving layers, but it can for scaling them, which we do here